src/app/views/app-sections/ResponseMessages.tsx

```diff
@@ -49,7 +49,7 @@ export function responseMessages(graphResponse: IGraphResponse, sampleQuery: IQu
   if (odataLink) {
     return (
       <MessageBar messageBarType={MessageBarType.info}>
-        <FormattedMessage id={'This response contains an @odata property'} />: @odata.{odataLink!.name}
+        <FormattedMessage id={'This response contains an @odata property.'} />: @odata.{odataLink!.name}
```

Comment:
Why was the full stop added? I don't think we should have it before a colon

Reply:
Looks like it is how it went into the all of the localization files